sdk/resourcemanager/Azure.ResourceManager/src/Generated/Resources/Models/ProviderExtendedLocation.Serialization.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Resources.Models
+{
+    public partial class ProviderExtendedLocation
```

Comment:
missing : IUtf8JsonSerializable?

Reply:
Keep the code as it is as explained in https://github.com/Azure/azure-sdk-for-net/pull/23091#discussion_r691937280